### PR TITLE
fix(extension): fix trezor integration and DApp tx signing with Trezor

### PR DESCRIPTION
**Pull request by tomislavhoracek**
# Checklist

- [x] https://input-output.atlassian.net/browse/LW-9071

---

## Proposed solution

This PR fixes DApp transaction signing using a Trezor hardware wallet. Also it fixes Trezor connection / communication in general since it was broken in the latest `main`

### CODE REVIEW

**Review by AngelCastilloB — APPROVED** [review]
LGTM @tomislavhoracek 🥇

File: apps/browser-extension-wallet/src/lib/wallet-api-ui.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import {
   walletManagerChannel,
   walletManagerProperties,
   walletRepositoryProperties
-} from '@cardano-sdk/web-extension';
+} from '../../../../node_modules/@cardano-sdk/web-extension/dist/cjs';
```
Comment (mkazlauskas) [inline apps/browser-extension-wallet/src/lib/wallet-api-ui.ts:14]: @tomislavhoracek please add any details you found about this to LW-9861

**Review by mkazlauskas — APPROVED** [review]
LGTM :rocket:

File: apps/browser-extension-wallet/src/lib/wallet-api-ui.ts (hunk 1 above)
Comment (tomislavhoracek) [reply apps/browser-extension-wallet/src/lib/wallet-api-ui.ts:14]: 👍

**Review by bslabiak — APPROVED** [review]

**Pull request merged**